### ISSUE TITLE
The train config of the models in torchvision

### ISSUE THREAD

**Issue by Ezra-Yu** [pytorch/vision#5150]
### 📚 The doc issue

Can you provide the configs of the provided  weights like https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/

for example, how do you train the regnet or shufflenet.

**Comment by NicolasHug** [comment]
Hi @Ezra-Yu . We're working on providing a more detailed documentation section about our models, and in particular we will try to include details on how each weights were trained. This will hopefully be available in the next release around Feb/March.

Meanwhile our training references are at https://github.com/pytorch/vision/tree/main/references/classification, although I'm personally not too familiar with the detailed training ref of regnet or shufflenet, as they might slightly differ from what you see in the current version. Perhaps this is something that @datumbox will know?

**Comment by datumbox** [comment]
@Ezra-Yu As Nicolas said, we are working to improve our documentation prior to the next release. 

For now the best way to find out how a specific model has been trained is by looking at the new model prototype API. Each model weight has a `recipe` URL field with a reference on how the weights were produced. 

For example here is the V1 of the weights of RegNet_Y_800MF:
https://github.com/pytorch/vision/blob/caff9d6d309b4c02dabdf46f2cd0c0c5e204c27b/torchvision/prototype/models/regnet.py#L92-L101

Here is the reference for V2:
https://github.com/pytorch/vision/blob/caff9d6d309b4c02dabdf46f2cd0c0c5e204c27b/torchvision/prototype/models/regnet.py#L102-L111

Each recipe link should contain information about the configuration of the model. Unfortunately this info might be missing for very old models contributed by the community. In such cases, we provide a reference to the PR that introduced them where you can read more info.

I'm going to close the issue as I believe my response covers your question but if it doesn't, feel free to open it again. Thanks!

**Comment by Ezra-Yu** [comment]
@datumbox, @NicolasHug 
Thank you very much for your detailed reply. Look forward to your next version!

**Comment by CA4GitHub** [comment]
> @Ezra-Yu As Nicolas said, we are working to improve our documentation prior to the next release.
> 
> For now the best way to find out how a specific model has been trained is by looking at the new model prototype API. Each model weight has a `recipe` URL field with a reference on how the weights were produced.
> 
> For example here is the V1 of the weights of RegNet_Y_800MF:
> 
> https://github.com/pytorch/vision/blob/caff9d6d309b4c02dabdf46f2cd0c0c5e204c27b/torchvision/prototype/models/regnet.py#L92-L101
> 
> Here is the reference for V2:
> 
> https://github.com/pytorch/vision/blob/caff9d6d309b4c02dabdf46f2cd0c0c5e204c27b/torchvision/prototype/models/regnet.py#L102-L111
> 
> Each recipe link should contain information about the configuration of the model. Unfortunately this info might be missing for very old models contributed by the community. In such cases, we provide a reference to the PR that introduced them where you can read more info.
> 
> I'm going to close the issue as I believe my response covers your question but if it doesn't, feel free to open it again. Thanks!

I don't see the `recipe` for [vision transformer models](https://github.com/pytorch/vision/blob/caff9d6d309b4c02dabdf46f2cd0c0c5e204c27b/torchvision/prototype/models/vision_transformer.py)
